dpc/src/testnet1/dpc.rs

```diff
@@ -93,51 +91,59 @@ impl DPCComponents for DPC {
 
     type InnerScalarField = <Self::InnerCurve as PairingEngine>::Fr;
     type OuterScalarField = <Self::OuterCurve as PairingEngine>::Fr;
-    
+    type OuterBaseField = <Self::OuterCurve as PairingEngine>::Fq;
+
     type AccountCommitment = PedersenCompressedCommitment<EdwardsBls12, 8, 192>;
     type AccountCommitmentGadget = PedersenCompressedCommitmentGadget<EdwardsBls12, Self::InnerScalarField, EdwardsBls12Gadget, 8, 192>;
-    
+    type AccountCommitmentOutput = <Self::AccountCommitment as CommitmentScheme>::Output;
```

Comment:
Explain: This redundant code is the cost we pay to remove the need to say "where" outside, in which we need to call out those types in the trait implementation.